client/index.html

```diff
@@ -1,9 +1,127 @@
+	<!-- // General meta information -->
+	
+	
+	<!-- Load Javascript -->
+	<script type="text/javascript" src="./js/jquery.js"></script>
+	<script type="text/javascript" src="./js/form.js"></script>
+	<script type="text/javascript" src="./js/jquery.query-2.1.7.js"></script>
+	<script type="text/javascript" src="./js/rainbows.js"></script>
+	<!-- // Load Javascipt -->
+
+	<!-- Load stylesheets -->
+	<link type="text/css" rel="stylesheet" href="css/style.css" media="screen" />
+	<!-- // Load stylesheets -->
+	
+<script>
+
+
+	$(document).ready(function(){
+ 
+	$("#submit1").hover(
+	function() {
+	$(this).animate({"opacity": "0"}, "slow");
+	},
+	function() {
+	$(this).animate({"opacity": "1"}, "slow");
+	});
+ 	});
+
+
+</script>
+	
+</head>
+<body>
+
+	<div id="wrapper">
+		<div id="wrappertop"></div>
+
+		<div id="wrappermiddle">
+
+			<h2 id="log">Login to Play!</h2>
+
+		<form id='myform' method='POST' target='formresponse'>
+			<div id="username_input">
+
+				<div id="username_inputleft"></div>
+
+				<div id="username_inputmiddle">
+					<input type="text" name="link" id="url" value="E-mail Address" onclick="this.value = ''">
+					<img id="url_user" src="./images/mailicon.png" alt="">
+				</div>
+
+				<div id="username_inputright"></div>
+
+			</div>
+
+			<div id="password_input">
+
+				<div id="password_inputleft"></div>
+
+				<div id="password_inputmiddle">
+					<input type="password" name="link" id="url" value="Password" onclick="this.value = ''">
+					<img id="url_password" src="./images/passicon.png" alt="">
+				</div>
+
+				<div id="password_inputright"></div>
+
+			</div>
+
+			<div id="submit">
+				<input type="image" src="./images/submit_hover.png" id="submit1" value="Sign In">
+				<input type="image" src="./images/submit.png" id="submit2" value="Sign In">
+			</div>
+		</form>
+			
+		<h2 id="register" style="display: none;">Sign Up to Play!</h2>
+		<div id="signUp" style="display: none;">
+			<form id='registerForm' method='POST' target='formresponse'>
+				<div id="username_signup">
+					<div id="username_inputleft"></div>
+						<div id="username_inputmiddle">
+							<input type="text" name="link" id="url" value="E-mail Address" onclick="this.value = ''">
+							<img id="url_user" src="./images/mailicon.png" alt="">
+						</div>
+					<div id="username_inputright"></div>
+				</div>
+				<div id="password_signup">
+					<div id="password_inputleft"></div>
+					<div id="password_inputmiddle">
+						<input type="password" name="link" id="url" value="Password" onclick="this.value = ''">
+						<img id="url_password" src="./images/passicon.png" alt="">
+					</div>
+					<div id="password_inputright"></div>
+				</div>
+				<div id="submit">
+					<input type="image" src="./images/submit_hover.png" id="submit1" value="Sign Up">
+					<input type="image" src="./images/submit.png" id="submit2" value="Sign Up">
+				</div>
+			</form>	
+		</div>
+			<div id="links_left">
+
+			<a href="#">Forgot Password</a>

```

Comment:
We are not allowing the user to recover password, since we're not accepting e-mail addresses.